src/Xamarin.Android.Tools/Sdks/MonoDroidSdkWindows.cs

```diff
@@ -7,17 +7,33 @@ namespace Xamarin.Android.Tools
 {
 	class MonoDroidSdkWindows : MonoDroidSdkBase
 	{
+		static readonly string[] VisualStudioPaths = new[] {
+			Environment.GetEnvironmentVariable ("VSINSTALLDIR"),
+			Path.Combine (OS.ProgramFilesX86, "Microsoft Visual Studio", "2017", "Enterprise"),
+			Path.Combine (OS.ProgramFilesX86, "Microsoft Visual Studio", "2017", "Professional"),
+			Path.Combine (OS.ProgramFilesX86, "Microsoft Visual Studio", "2017", "Community"),
+		};
+
 		protected override string FindRuntime ()
 		{
 			string monoAndroidPath = Environment.GetEnvironmentVariable ("MONO_ANDROID_PATH");
 			if (!string.IsNullOrEmpty (monoAndroidPath)) {
 				if (Directory.Exists (monoAndroidPath) && ValidateRuntime (monoAndroidPath))
 					return monoAndroidPath;
 			}
+
 			string xamarinSdk = Path.Combine (OS.ProgramFilesX86, "MSBuild", "Xamarin", "Android");
```

Comment:
There's another problem here: `xamarinSdk` is the VS 2015 install location. `VisualStudioPaths` contains VS 2017 locations.

It is possible to have *both* VS 2015 and VS 2017 installed.

Should we really prefer the VS 2015 install location over the VS 2017 install location, if both are installed? I think we should check for VS 2015 *after* VS 2017 locations.

Additionally, this could be cleaned up by "moving" `xamarinSdk` into `VisualStudioPaths`, so that they're treated uniformly.